cmd/vcluster/context/flag_options.go

```diff
@@ -35,11 +35,11 @@ type VirtualClusterOptions struct {
 
 	SetOwner bool `json:"setOwner,omitempty"`
 
-	SyncAllNodes        bool `json:"syncAllNodes,omitempty"`
-	EnableScheduler     bool `json:"enableScheduler,omitempty"`
-	DisableFakeKubelets bool `json:"disableFakeKubelets,omitempty"`
-
-	TranslateImages []string `json:"translateImages,omitempty"`
+	SyncAllNodes        bool     `json:"syncAllNodes,omitempty"`
```

Comment:
I just realized this format change happened, happy to revert this back if you prefer.